src/kernels/installer/productInstaller.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable max-classes-per-file */
+
+import { Memento, Uri } from 'vscode';
+import { ProductNames } from './productNames';
+import { Product } from './types';
+import { PythonEnvironment } from '../../platform/pythonEnvironments/info';
+import { InterpreterUri } from '../../platform/common/types';
+import { isResource } from '../../platform/common/utils/misc';
+import { getInterpreterHash } from '../../platform/pythonEnvironments/info/interpreter';
+
+/**
+ * Keep track of the fact that we attempted to install a package into an interpreter.
+ * (don't care whether it was successful or not).
+ */
+export async function trackPackageInstalledIntoInterpreter(
```

Comment:
These loose functions didn't have any .node dependencies but were used elsewhere so moved them into a separate file.

Reply:
Sometimes those are helpers.ts? Or does that note fit here?